share/data/Sicile.mtg

```diff
@@ -4235,4237 +4235,3 @@ ENTITY-CODE				Experiment_name	Sample_date	Architecture_date	Genotype	Modality	P
 	+A																														
 	^/bt																														
 ^<bt										18																					
-=======
```

Comment:
Are you sure the file has been duplicated for Sicily? 
If I accept the PR, won't we lose this data? Because it is important and can be used for the next post-doc

Reply:
Where does this conflict come from? Do you know?

Reply:
In fact you remove a line in Silice 
Something like <<<<<<<<<<<<<<< HEAD
At the begining of the file.
Hoowever, when this appear you have both the previous and the next version in the same file.
I just remove this duplicate and the conflict strings.
So I just keep the last version.
We do not loose anything but just keep a valid file

Reply:
For more info see this commit:
https://github.com/openalea-incubator/strawberry/commit/41cca43880d34b2d4875e9e562f0cf9de97a09a2